action/stakecreate_test.go

```diff
@@ -28,6 +28,8 @@ var (
 	autoStake  = true
 	index      = uint64(10)
 	senderKey  = identityset.PrivateKey(27)
+	zero       = "0"
+	negtive    = "-10"
```

Comment:
again: it is a bad practice to have some global parameters with such common names for unit test purpose.

Reply:
suggest to change this to a table-based test

var testActionParams = []struct{
  gaslimit   uint64
  gasprice *big.Int
  ...
  expected error
} {
  // valid test
  {
     ...
     expected: nil,
  }
  // invalid test
  {
     ...
     expected: ErrInvalidAmount,
  }
}

also with this you can do pass/fail test at same time, no need to add TestNotPositive()

Reply:
done